tsconfig.json

```diff
@@ -10,8 +10,9 @@
     "declaration": true,
     /* Strict Type-Checking Options */
     "strict": true /* Enable all strict type-checking options. */,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "esModuleInterop": true, /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "skipLibCheck": true
```

Comment:
I'm somewhat doubtful that the upstream maintainers will be okay with enabling `skipLibCheck`